packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -251,7 +251,7 @@ const getMetaStyles = (theme: GrafanaTheme2) => ({
     width: '100%',
     fontSize: theme.typography.size.sm,
     color: theme.colors.text.secondary,
-    margin: theme.spacing(0.5, 0, 0),
+    margin: `${theme.spacing.x50} 0 0`,
```

Comment:
The `theme.spacing()` function has an advantage that it works the same as CSS `margin` property with the shorthands, while this version seems to be unnecessarily verbose.